modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 6.0.1 - TBD
 
+### Changed
+- The option`end_container_on_empty_block` can now take a string of blocks to split when pressing Enter twice #TINY-6559
+- The default value for option`end_container_on_empty_block` changed to `blockquote` #TINY-6559
```

Comment:
This should be in the `Unreleased` section of the changelog since it's for 6.1.0 instead of 6.0.1.

- [ ] Move to correct place

Also here's a suggestion for some minor edits:
```suggestion
### Changed
- The `end_container_on_empty_block` option can now take a string of blocks to split when pressing Enter twice #TINY-6559
- The default value for `end_container_on_empty_block` option has been changed to ` 'blockquote' ` #TINY-6559
```